frameworks/FSharp/giraffe/src/App/Custom.fs

```diff
@@ -11,6 +11,7 @@ open System.IO
 
 let private DefaultCapacity = 1386
 let private MaxBuilderSize = DefaultCapacity * 3
+let private BufferSize = 27

```

Comment:
That's a magic number?

Reply:
Yes, unfortunately. I copied it from here: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/CSharp/aspnetcore/Benchmarks/Middleware/JsonMiddleware.cs#L20

Reply:
Cc @sebastienros